spec/services/metric_builder_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+RSpec.describe MetricBuilder do
+  describe "#run_all" do
+    let(:metrics) { { m1: 1 } }
+
+    it "calls each metric class once" do
+      expect_any_instance_of(Metrics::NumberOfPdfsMetric).to receive(:run).exactly(1).times.and_return(metrics)
+
+      subject.run_all({})
+    end
+
+    it "returns a merged hash of metrics" do
+      allow_any_instance_of(Metrics::NumberOfPdfsMetric).to receive(:run).and_return(metrics)
+      result = subject.run_all({})
+
+      expect(result).to eq(metrics)
```

Comment:
I'm not sure what this test is doing. The return value from `Metrics::NumberOfPdfsMetric` is being mocked, so it's not necessary to test the value.